localstripe/resources.py

```diff
@@ -2878,3 +2878,67 @@ def __init__(self, card=None, customer=None, **kwargs):
 
         self.type = 'card'
         self.card = card_obj
+
+
+class Balance(StripeObject):
+    object = 'balance'
+    # Balance does not have id prefix
+    _id_prefix = 'balance_'
+
+    def __init__(self, **kwargs):
+        if kwargs:
+            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
+
+        _currencies = ["usd", "eur"]
+        avaliable, pending = [], []
+
+        # By default there are neither avaliable, nor pending funds
+        for currency in _currencies:
+            avaliable.append({"amount": 0, "currency": currency})
+            pending.append({"amount": 0, "currency": currency})
+
+        # Set the data in the Stripe specification
+        # https://stripe.com/docs/api/balance/balance_object
+        avaliable_obj = [
+            self._get_fund(amount=a["amount"], currency=a["currency"])
+            for a in avaliable
+        ]
+
+        pending_obj = [
+            self._get_fund(amount=p["amount"], currency=p["currency"])
+            for p in pending
+        ]
+
+        # All exceptions must be raised before this point.
+        super().__init__(id="")
```

Comment:
I propose:
```
# All exceptions must be raised before this point.
super().__init__(id="balance")
```
I think it make more sense.
Also, it should be move just after 
```
        if kwargs:
            raise UserError(400, 'Unexpected ' + ', '.join(kwargs.keys()))
        # All exceptions must be raised before this point.
        super().__init__(id="balance")
```
because (I'm not 100% sure) the other objects call it as soon as possible (just after all API arguments been checked)